_i18n/de/resources/moneropedia/tail-emission.md

```diff
@@ -1,15 +1,15 @@
 ---
-terms: ["Tail-Emission"]
-summary: "the block reward at the end of the emission curve"
+terms: ["Tail-Emission", "Anschlussverteilung"]
+summary: "Die festgesetzte Blockbelohnung am Ende der Verteilungskurve"
```

Comment:
```suggestion
summary: "Die festgesetzte Blockbelohnung am Ende der Emissionskurve"
```

I'd stick to "Emission" in German too, see https://de.wikipedia.org/wiki/Emission_(Wirtschaft)

Reply:
While your suggestion would be completely correct, I am going to use the agreed terms of the terminology guide for now.